README.md

```diff
@@ -1,3 +1,20 @@
-# Example Subgraph
+# Validator Rewards Example
 
-An example to help you get started with The Graph. For more information see the docs on https://thegraph.com/docs/.
+This example subgraph stores `Reward` objects that represent rewards received by a validator in the Cosmos Hub chain. In order to do that, an event handler is used to filter [reward](https://github.com/cosmos/cosmos-sdk/blob/main/x/distribution/spec/06_events.md) events. The type of event to be filtered is specified in the subgraph manifest file. That way, the handler will just receive events of that type.
```

Comment:
Let's pin point a specific commit in the Cosmos SDK link:
https://github.com/cosmos/cosmos-sdk/blob/13378bd2cfb9695da6477494e449b0a3bca9bc94/x/distribution/spec/06_events.md

That way if they change anything on the main branch in the future, this will still be available.